### PR TITLE
Fix: Formatting issues in docs

### PR DESCRIPTION
**Pull request by yo1995**
With #919  merged, this PR runs automated check on docs and fixed minor issues.

Will either setup sanity checks in our repo in next release, or periodically check docs formatting in the future.

### CODE REVIEW

File: arcgis-ios-sdk-samples/Edit data/Edit features with feature-linked annotation/README.metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "category": "Edit data",
-    "description": "Edit feature attributes which are linked to annotation through an expression.",
+    "description": "Edit feature attributes which are linked to annotations through an expression.",
```
Comment (vquach2404) [inline arcgis-ios-sdk-samples/Edit data/Edit features with feature-linked annotation/README.metadata.json:3]: If you're going to make this change, please also update the description in content plist.

Comment (yo1995) [reply arcgis-ios-sdk-samples/Edit data/Edit features with feature-linked annotation/README.metadata.json:3]: Ahh really great catch! So which way should we go? 
In README it is "annotations" while in metadata and content plist it is "annotation".

Comment (philium) [reply arcgis-ios-sdk-samples/Edit data/Edit features with feature-linked annotation/README.metadata.json:3]: @vquach2404 Good catch. I just assumed this was fixing a discrepancy between the two.

**Comment by yo1995** [comment]
👆  Thanks for pointing this out! I'll add a check to the script to ensure the description strings are also consistent.

File: arcgis-ios-sdk-samples/Edit data/Edit features with feature-linked annotation/README.metadata.json (hunk 1 above)
Comment (vquach2404) [reply arcgis-ios-sdk-samples/Edit data/Edit features with feature-linked annotation/README.metadata.json:3]: @yo1995 let's just change the metadata and plist to "annotations." I originally adjusted the README to use "annotations" instead.

**Comment by yo1995** [comment]
I've added the check and fixed more inconsistent plist descriptions! 
Except a few made inconsistent on purpose, now all the descriptions are the same in README, plist and metadata.

**Pull request merged**